### PR TITLE
mod : speed glitch fix

### PR DESCRIPTION
**Pull request by namidaka**

### CODE REVIEW

**Comment by namidaka** [comment]
solves #1153

File: src/Module.Server/Common/Models/CrpgAgentStatCalculateModel.cs [hunk 1]
```diff
@@ -448,7 +448,7 @@ private float ImpactofStrAndWeaponLengthOnTimeToMaxSpeed(int weaponLength, int s
 
     private int MaxWeaponLengthForStrLevel(int strengthSkill)
     {
-        int uncappedMaxWeaponLength = (int)(45 + (strengthSkill - 3) * 9 + Math.Pow((strengthSkill - 3) * 0.14677993f, 12));
+        int uncappedMaxWeaponLength = (int)(45 + (strengthSkill - 3) * 9 + Math.Pow(Math.Min(strengthSkill - 3, 24) * 0.14677993f, 12));
```
Comment (verdie-g) [inline src/Module.Server/Common/Models/CrpgAgentStatCalculateModel.cs:451]: Does that need to updated in the UI?

**Review by verdie-g — CHANGES_REQUESTED** [review]

File: src/Module.Server/Common/Models/CrpgAgentStatCalculateModel.cs (hunk 1 above)
Comment (namidaka) [reply src/Module.Server/Common/Models/CrpgAgentStatCalculateModel.cs:451]: no.
image
this is with 54 str

Comment (verdie-g) [reply src/Module.Server/Common/Models/CrpgAgentStatCalculateModel.cs:451]: Even if js has a dumb behavior for overflows it's better to keep the code aligned. I'll do the commit.

**Review by verdie-g — APPROVED** [review]

**Pull request merged**